cmd/allinone/main.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"kubesphere.io/devops/cmd/allinone/app"
+	"log"
+)
+
+func main() {
+	cmd := app.NewCommand()
+	if err := cmd.Execute(); err != nil {
+		log.Fatalln(err)
```

Comment:
Need append `os.Exit(1)` below here.

Reply:
It's ok if it returns an error code once the command line exit with an error.

![image](https://user-images.githubusercontent.com/1450685/124096864-929def80-da8d-11eb-9d6c-9d57724fd133.png)